lib/mi_bridges/driver/more_about_job_income_page.rb

```diff
@@ -57,7 +65,7 @@ def fill_in_salary
       end
 
       def current_member
-        members.select do |member|
+        @current_member ||= members.select do |member|
```

Comment:
`@_current_member` to keep up with convention?

Reply:
good catch! 🐟  done at  060d4b2